src/LGPL-2.0-or-later.xml

```diff
@@ -5,21 +5,6 @@
     <crossRefs>
       <crossRef>http://www.gnu.org/licenses/old-licenses/lgpl-2.0-standalone.html</crossRef>
     </crossRefs>
-    <standardLicenseHeader>
-      Copyright (C)<alt name="copyright" match=".+">year name of author</alt>
-      <p>This library is free software; you can redistribute it and/or modify it
-      under the terms of the GNU Library General Public License as published
-        by the Free Software Foundation; version 2 or any later version.</p> 
```

Comment:
This differs from the FSFs recommended “any later version” wording from `<text>`.  `LGPL-2.1-or-later` had the same issue.